### ISSUE TITLE
git send-email patches mangled by `Content-Transfer-Encoding: quoted-printable`

### ISSUE THREAD

**Issue by zamaudio** [ProtonMail/proton-bridge#461]
I am using git send-email to mail code patches to a mailing list.
Our maintainer uses plain mbox format to read patch emails.
Therefore, we need the transfer encoding header to be preserved so that
the patch can flow unmangled.  

## Expected Behavior
Content-Transfer-Encoding header should be preserved if set.

## Current Behavior
Content-Transfer-Encoding is always set to quoted-printable.

## Possible Solution
Remove the line of code that overwrites the header?
Or make it detect some values of X-Mailer and preserve the header if that is set.

## Version Information
Proton Mail Bridge 3.8.2 with git send-email 2.30.2

**Comment by LBeernaertProton** [comment]
Hey @zamaudio thanks for the report. Could you provide us with a sample message that arrives on Bridge's SMTP server so we can investigate further?

You can capture the SMTP input with:

```
bridge -l=trace --log-smtp > capture.log
```

Note that the above log will contain sensitive information, it's advised to delete everything but the SMTP when submitting.

**Comment by ethancedwards8** [comment]
I have confirmed that this is my issue as well. Mutt sets my encoding to 7bit, but that is changed by the protonmail-brdige to quoted-printable.

**Comment by ethancedwards8** [comment]
The issues are: https://github.com/ProtonMail/proton-bridge/blob/60633fc09ccf530dd9ae50758722409af2ccdbbd/pkg/message/parser/parser.go#L112

and https://github.com/ProtonMail/proton-bridge/blob/60633fc09ccf530dd9ae50758722409af2ccdbbd/pkg/message/build.go#L495

**Comment by ethancedwards8** [comment]
I will be leaving protonmail if these issues are not resolved. This is a technical and ethical problem.

**Comment by exander77** [comment]
@ethancedwards8 I am personally leaving ProtonMail.

**Comment by ethancedwards8** [comment]
@exander77 yes. I decided earlier today to migrate to mailbox.org. Have any ideas of moving all of my emails between the services?

**Comment by exander77** [comment]
@ethancedwards8 I generally use my own tool: https://github.com/exander77/mailtool.py
But it is really crude. Inspired by imapsync, so maybe better use that.
